src/App.js

```diff
@@ -1,16 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
 const App = () => {
+  const [entries, setEntries] = useState(chatMessages);
+  const [likes, setLikes] = useState(0);
```

Comment:
Because we can determine the number of likes from `entries`, `likes` should not be its own state variable. In general, we want state variables to not depend on one another, otherwise we get multiple sources of truth and increase the risk of bugs.